examples/rln-js/index.html

```diff
@@ -308,7 +308,7 @@ <h4 class="mu1">Messages</h4>
         }
       };
 
-      const rlnDeployBlk = 7109391;
+      const rlnDeployBlk = 8813000;
       const rlnAddress = "0x4252105670fe33d2947e8ead304969849e64f2a6";
```

Comment:
Block is changed but not address?

Reply:
Forgot to bring this topic.
There is a problem with Goerli, I suppose that because block are produced really fast - gateway just fires error.

![image](https://user-images.githubusercontent.com/118575614/233177740-8841a0a8-456c-4a64-a8a0-27163469c3ec.png)

So in order to retrieve contract state we need to update start block. I think I can write something to always fetch last ~500 blocks but not sure if we need it.

Reply:
hm.. this sounds problematic. We might need to retrieve the blocks in different batches of N blocks, otherwise we risk not getting all the membership register events. Which function is returning the error? this?
```js
const registeredMemberEvents = await this.contract.queryFilter(
      this.membersFilter,
      fromBlock
    );
```

Reply:
@richard-ramos exactly
I suppose another way to fix it would be to query in smaller chunks of blocks by `queryFilter`.

Do you think it's worth doing, @richard-ramos @fryorcraken ? I can follow-up with a util function for js-rln

Reply:
IMO it should be the way to go, unless there's a different way to retrieve logs.
cc: @rymnc because this might be an issue that can be happening in nwaku (and go-waku) too and require fixing!

Reply:
Yes @richard-ramos this has been on my mind for a while now. We dont have an issue with this at the moment, but before we move to mainnet we need to be chunking the queries.

Reply:
Addressed it here - https://github.com/waku-org/js-rln/pull/57